src/sass/components/_intro-modal.scss

```diff
@@ -124,6 +124,7 @@ $backdrop-border-radius: 0 0 8px 8px;
 
   &__button-wrap {
     width: 100%;
+    margin-bottom: 2em;
```

Comment:
Note, I got an eslint error for this line. I will fix this, but please make sure to double-check checkins against `npm test`:
src/sass/components/_intro-modal.scss
 127:5  ✖  Expected variable for "margin-bottom".   sh-waqar/declaration-use-variable